pay-api/src/pay_api/services/invoice_search.py

```diff
@@ -512,43 +514,21 @@ def get_invoices_totals(invoices: dict, statement: dict) -> dict:
     @user_context
     def generate_payment_report(report_inputs: PaymentReportInput, **kwargs):  # pylint: disable=too-many-locals
         """Prepare data and generate payment report by calling report api."""
-        labels = [
-            "Product",
-            "Corp Type",
-            "Transaction Type",
-            "Transaction",
-            "Transaction Details",
-            "Folio Number",
-            "Initiated By",
-            "Date",
-            "Purchase Amount",
-            "GST",
-            "Statutory Fee",
-            "BCOL Fee",
-            "Status",
-            "Corp Number",
-            "Transaction ID",
-            "Invoice Reference Number",
-        ]
-
         content_type = report_inputs.content_type
         results = report_inputs.results
         report_name = report_inputs.report_name
         template_name = report_inputs.template_name
 
-        # Use the status_code_description instead of status_code.
-        invoice_status_codes = CodeService.find_code_values_by_type(Code.INVOICE_STATUS.value)
-        for invoice in results.get("items", None):
-            filtered_codes = [cd for cd in invoice_status_codes["codes"] if cd["code"] == invoice["status_code"]]
-            if filtered_codes:
-                invoice["status_code"] = filtered_codes[0]["description"]
-
         if content_type == ContentType.CSV.value:
-            template_vars = {
-                "columns": labels,
-                "values": InvoiceSearch._prepare_csv_data(results),
-            }
```

Comment:
I talked to Jia about this, not sure if it makes sense to call `report-api` for something `pay-api` could just simply return